### PR TITLE
Display log messages to IDE output

### PR DESCRIPTION
**Pull request by smadala**
- Displays log messages to ide test output window.
- When multiple source passed, If there is abort due to one source, test run/discovery should continue for rest of sources.

Relates issue: #281 

Test Plan:
- Follow the issue repro instructions. able to see error message in IDE test output window.
- Open TestPlatform.sln in patched VS IDE, discover and run all project test.

### CODE REVIEW

File: src/Microsoft.TestPlatform.CommunicationUtilities/EventHandlers/TestDiscoveryEventHandlerExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                LastDiscoveredTests = null,
+                TotalTests = -1
+            };
+            rawMessage = dataSerializer.SerializePayload(MessageType.DiscoveryComplete, payload);
+            discoveryEventsHandler.HandleRawMessage(rawMessage);
+
+            // Complete discovery.
+            discoveryEventsHandler.HandleDiscoveryComplete(-1, null, true);
+        }
+
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.CommunicationUtilities/EventHandlers/TestDiscoveryEventHandlerExtensions.cs]: nit: remove extra line

File: src/Microsoft.TestPlatform.CrossPlatEngine/Client/Parallel/ParallelProxyExecutionManager.cs [hunk 2]
```diff
@@ -155,7 +155,7 @@ public void Close()
 
             // In Case of Cancel or Abort, no need to trigger run for rest of the data
             // If there are no more sources/testcases, a parallel executor is truly done with execution
-            if (testRunCompleteArgs.IsAborted || testRunCompleteArgs.IsCanceled || !this.StartTestRunOnConcurrentManager(proxyExecutionManager))
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.CrossPlatEngine/Client/Parallel/ParallelProxyExecutionManager.cs]: Please explain this change. And also fix the comments.

File: src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs [hunk 3]
```diff
@@ -11,6 +11,7 @@ namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities
 
     using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.Interfaces;
     using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.ObjectModel;
+    using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.EventHandlers;
```
Comment (harshjain2) [inline src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs]: Namespaces are not sorted in correct order.

File: src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyDiscoveryManager.cs [hunk 4]
```diff
@@ -10,6 +10,7 @@ namespace Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Client
     using Microsoft.VisualStudio.TestPlatform.Common.ExtensionFramework;
     using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities;
     using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.Interfaces;
+    using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.EventHandlers;
```
Comment (harshjain2) [inline src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyDiscoveryManager.cs]: nit : sort namespaces.

**Comment by singhsarab** [comment]
These changes have not been approved. Closing the pull request.

**Pull request closed without merging**